src/client/hud.cpp

```diff
@@ -841,7 +841,13 @@ void Hud::drawSelectionMesh()
 			u32 b = (selectionbox_argb.getBlue() *
 					m_selection_mesh_color.getBlue() / 255);
 			driver->draw3DBox(box, video::SColor(255, r, g, b));
+
+			if(g_settings->getBool("node_light")){
+				v3f xcolor = g_settings->getV3F("node_light_color");
+				driver->draw3DBox(selection_box, video::SColor(255, xcolor.X, xcolor.Y, xcolor.Z));
```

Comment:
why does this use `selection_box` instead of `box` like above?

Reply:
Code editor error